content/docs/custom-plugins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Installing Custom Plugins
+lastUpdated: '2021-03-16T17:55:00.0Z'
+description: How to add a custom Backstage plugin to Roadie
+---
+
+# Introduction
+
+Your plugins can be installed via npm or yarn, like publishing a normal package. We will provide you a unique scope for
+your plugins, and we host a dedicated npm registry which you can publish to.
+
+## Prerequisites
+
+- For our alpha release you'll need to give us some information upfront about your plugin. We plan to make all of this
+  self-service going forward but currently backstage doesn't support this out-of-the-box. For now, we will need you to
+  fill out [this form][form].
+
+- Get your credentials for the npm registry. Follow [this link][forgot-password] to reset your password. The username is
+  `<your-company>-roadie`. This will send a reset link to the email we have associated with your account. You can
+  change this email as you wish.
+
+- Configure npm. There are two options:
+
+  - Interactively using npm login. Suitable for a developer testing the process.
+    <pre>
```

Comment:
You'll get code formatting and syntax highlighting here if you use backticks with the name of the language.

Reply:
I used `<pre>` because I want to highlight in bold the test that needs to be replaced. Should I change it back and remove the highlighting?

![pre_bold_highlight](https://user-images.githubusercontent.com/736631/111458137-5bdbdd80-8719-11eb-9ba5-642334c57bf3.png)

with backticks:
![backticks](https://user-images.githubusercontent.com/736631/111458498-c725af80-8719-11eb-9ab6-e5a3fb8909ca.png)

Reply:
I guess that makes sense. I have opened https://github.com/RoadieHQ/marketing-site/issues/142 for more work in this area.